src/main/twirl/com/lightbend/lagom/docs/getstartedjava.scala.html

```diff
@@ -52,48 +79,66 @@
                     </ul>
 
                     <p>
-                        See <a href="@context.path/documentation/@context.currentDocsVersion/java/JavaPrereqs.html">detailed instructions</a>
-                        for help setting up the prerequisites and getting started. Or, if you are raring to go, just follow these simple steps:
+                        If you have the prerequisites installed and are familiar with Maven, 
+                        create a Hello World project using the commands below. Otherwise, see <a href="@context.path/documentation/@context.currentDocsVersion/java/JavaPrereqs.html">detailed instructions</a>
+                        for help setting up the prerequisites and getting started.
                     </p>
 
                     <ol>
                         <li>
-                            <p>
-                                Create a new project from the command line (all on one line):
-                            </p>
+                          
+                                From a command prompt with administrative privileges, create a new project by entering the following (all on one line):
```

Comment:
I tried it out on a fresh Windows virtual machine, and I didn't need to run the command prompt as an administrator to execute this Maven command. I haven't tried sbt yet.